### PR TITLE
Update tox version

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

**Comment by kurnevsky** [comment]
License should be changed for that :)

File: Cargo.toml [hunk 1]
```diff
@@ -35,27 +35,28 @@ maintainer-scripts = "dpkg/info"
...
-futures = "0.1"
+futures = { version = "0.3", default-features = false, features = ["std", "async-await"] }
 hex = "0.3"
 itertools = "0.8"
 log = "0.4"
 regex = "1"
 serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.8"
-tox = "0.0.10"
+tox = { version = "0.1.0", git = "https://github.com/tox-rs/tox.git" }
```
Comment (kpp) [inline Cargo.toml:48]:
```suggestion
tox = { version = "0.1.0" }
```

File: Cargo.toml [hunk 2]
```diff
@@ -35,27 +35,28 @@ maintainer-scripts = "dpkg/info"
...
+tox_crypto = { version = "0.1.0", git = "https://github.com/tox-rs/tox.git" }
```
Comment (kpp) [inline Cargo.toml:49]: Why don't we reexport crypto from tox crate?

Comment (suhr) [reply Cargo.toml:49]: Because we forgot to add `tox_crypto` into `tox` crate.

**Pull request merged**